src/scripts/modules/media/tabs/contents/toc/section.coffee

```diff
@@ -82,8 +82,11 @@ define (require) ->
       if e.keyCode is 13 or e.keyCode is 32
         e.preventDefault()
         @toggleSection()
-        @$el.find('> div > .section-wrapper').focus()
-        @$el.find('> ul').attr('role', 'alert')
+        isExpanded = @$el.find('button')[0].getAttribute('aria-expanded')
+        if isExpanded == 'true'
+          @$el.find('ul')[0].querySelector('.name-wrapper a, .section-wrapper').focus()
+        else if isExpanded == 'false'
```

Comment:
Is there an `else` case or does this code assume that the `aria-expanded` attribute is _always_ present on the button? (I jut worry about accidental code fall-through) 

Maybe something like:

```coffeescript
if isExpanded is 'true'
    focusEl = @$el.find('ul')[0].querySelector('.name-wrapper a, .section-wrapper')
else
    focusEl = @$el.find('> div > .section-wrapper')[0]

focusEl.focus()
```

Reply:
@philschatz Actually this function is called only on element which have `aria-attribute` but your version looks better :D